spec/requests/feed_request_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Feeds', type: :request do
+  # authorization
+  let!(:user) { create(:user) }
+  let!(:auth_headers) { user.create_new_auth_token }
+  let!(:headers) do
+    {
+      'uid' => auth_headers[:uid],
+      'access-token' => auth_headers['access-token'],
+      'client' => auth_headers[:client]
+    }
+
+    # database population
+  end
+  let!(:communication) { create_list(:communication, 200) }
+
+  describe 'GET api/v1/feed' do
+    context 'with authorization' do
+      it 'lists the events' do
+        get '/api/v1/feed', headers: auth_headers
+        expect(response).to have_http_status 200
+      end
```

Comment:
You are not checking that this lists the events, there aren't any events in the database 🤔. You should check that there are communications and that they are returned correctly, no need to test this with 200 objects